TransIT.API/Controllers/StateController.cs

```diff
@@ -26,19 +26,19 @@ IODCrudService<State> odService
         [HttpPost]
```

Comment:
Whole crud must be allowed only for admin. For others readonly.

Reply:
done